pylint_django/tests/input/migrations/0003_without_backwards.txt

```diff
@@ -0,0 +1 @@
+missing-backwards-migration-callable:12:Migration:pylint_django.tests.input.migrations.0003_without_backwards Always include backwards migration callable
```

Comment:
The error in CI that you are seeing with this test is that the txt file tells it to expect the error on line 12 while the RunPyhon node which gets inspected and then used for error annotations in on line 13. See the log message:
```
Unexpected in testdata:

E             13: missing-backwards-migration-callable
```

Reply:
Can't believe I missed that. I wasn't paying attention to that, don't know why I was fixated on 12. Thank you

Reply:
Still having the same problem for some reason.

Reply:
Every line in the test input file which is doing to trigger an error also needs to be annotated with a comment to tell the test suite what error message is expected, see `0002_new_column.py`. Then this is matched against the actual messages received from the linter.